### PR TITLE
Weld 3.0.0.CR1

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: news/2017-01-23-weld-300CR1.asciidoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+desc: Weld 3.0.0.CR1 released
+tags: [release]
+---
+:linkattrs:
+
+I am very pleased to announce the release of Weld *3.0.0.CR1* (aligned with *CDI 2.0-PFD*).
+See also https://issues.jboss.org/projects/WELD/versions/12331714[the release details, window="_blank"].
+Thanks to everyone involved in this release!
+This version of Weld will be part of the JSR 365 Approval Ballot (watch http://cdi-spec.org/ for news).
+The first release candidate is mostly dedicated to cleanup and minnor optimizations.
```
Comment (tremes) [inline news/2017-01-23-weld-300CR1.asciidoc]: Double `n` in minor.

File: news/2017-01-23-weld-300CR1.asciidoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+excerpt: Weld 3.0.0.CR1 (CDI 2.0-PFD) released. And it's almost Final!
+desc: Weld 3.0.0.CR1 released
+tags: [release]
+---
+:linkattrs:
+
+I am very pleased to announce the release of Weld *3.0.0.CR1* (aligned with *CDI 2.0-PFD*).
+See also https://issues.jboss.org/projects/WELD/versions/12331714[the release details, window="_blank"].
+Thanks to everyone involved in this release!
+This version of Weld will be part of the JSR 365 Approval Ballot (watch http://cdi-spec.org/ for news).
```
Comment (manovotn) [inline news/2017-01-23-weld-300CR1.asciidoc]: I would turn this into asciidoc link:
http://cdi-spec.org/ -> http://cdi-spec.org/[CDI spec]

File: news/2017-01-23-weld-300CR1.asciidoc (hunk 1 above)
Comment (manovotn) [inline news/2017-01-23-weld-300CR1.asciidoc]: minnor -> minor

**Comment by mkouba** [comment]
@tremes @manovotn Is it ok now?

**Comment by tremes** [comment]
yes

**Comment by mkouba** [comment]
Merged.

**Pull request closed without merging**